lib/fog/mock/errors.rb

```diff
@@ -0,0 +1,5 @@
+module Fog
+  module Errors
+    class MockNotImplemented < Fog::Errors::Error; end
+  end
+end
```

Comment:
I understand the practice and I believe I'm respecting it. Here the path is `fog/mock/errors.rb` for a `Fog::Mock::Errors` module. This module contains only one class for now. Let me put it this way: we don't want to create a file for each Error class.